lib/home_page.dart

```diff
@@ -0,0 +1,314 @@
+import 'package:flutter/material.dart';
+import 'question_model.dart';
+import 'category_filter.dart';
+
+class Home extends StatefulWidget {
+  @override
+  HomeState createState() => HomeState();
+}
+
+class HomeState extends State<Home> {
```

Comment:
Not as important, but can you rename `HomeState` to `_HomeState`

Reply:
Sorry, forgot to change that, but we can do it later

Reply:
Please fix this when you fix any merge conflicts.